src/mcts/search.cc

```diff
@@ -718,17 +725,29 @@ std::pair<Node*, bool> Search::PickNodeToExtend(Node* node,
   }
 }
 
-std::pair<Move, Move> Search::GetBestMove() const {
+std::tuple<Move, Move, int> Search::GetBestMove() const {
```

Comment:
> maybe it's better to move this setting out of search.cc into tournament.cc or game.cc, and instead of GetBestMove() 



Not sure how to do this cleanly w/regards to getting an OptionsParser in game.cc. I'll take a look tomorrow, but if you can offer any tips or code for that, I'd be super grateful!